src/model/Simulation.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+from dataclasses import dataclass
+from typing import Any, List
+
+from model.car import Car
+from model.geom.track import Track
+
+
+@dataclass
+class Simulation:
+    track: Track
+    ai: Any = None
+
+    def get_positions(self) -> List[Car]:
```

Comment:
get_positions should return positions, not cars

Reply:
Final version of naming will be used when simulation class will be available not just its stub